lti_consumer/lti_xblock.py

```diff
@@ -93,6 +93,7 @@
     'instructor': 'Instructor',
 }
 CUSTOM_PARAMETER_TEMPLATE_TAGS = ('${', '}')
+CUSTOM_PARAMETER_REGEX = re.compile(r'^( *\w+ *\= *( *\w+ *|\$\{ *\w+ *\}) *)$')
```

Comment:
With this regex something like 'x="y"' is not valid, is that desirable?

Reply:
@alexjmpb I don't think 'x="y"' should be necessary since the key and value is already contained in a string, having "y" would be redundant and will require a more complex regex